go-console/.conf/checkDeps.ps1

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
This file can be removed, this is automatically copied from scripts to the project folder during the creation.